src/Database/Snowflake/SnowflakeDatabaseReflection.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TableBackendUtils\Database\Snowflake;
+
+use Doctrine\DBAL\Connection;
+use Keboola\TableBackendUtils\Database\DatabaseReflectionInterface;
+use Keboola\TableBackendUtils\DataHelper;
+use Keboola\TableBackendUtils\Escaping\Snowflake\SnowflakeQuote;
+
+final class SnowflakeDatabaseReflection implements DatabaseReflectionInterface
+{
+    private Connection $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getUsersNames(?string $like = null): array
+    {
+        $likeSql = '';
+        if ($like !== null) {
+            $likeSql .= sprintf(
+                ' LIKE %%%s%%',
```

Comment:
Tohle musí udělat `LIKE %'escaped'%`, ne? 

Ty procenta by měly být v ::quote()

Reply:
03-21e2-0000-12e50626f606
SHOW ROLES LIKE %'SUPER'%

Reply:
No tak samozřejmně :facepalm: nevím co za vola to napsalo, dík.